src/config/thorough.ts

```diff
@@ -26,7 +26,7 @@ export const overridesWhenThorough = (function () {
       },
       rules: {
         "@typescript-eslint/await-thenable": "error",
-        "@typescript-eslint/no-floating-promises": "error",
+        "no-floating-promises-qawolf": "error",
```

Comment:
```suggestion
        "no-floating-promises": "error",
```

@denis-sokolov Given that we're using `reword`, maybe it's cleaner to use the original name.

Reply:
The name of the rule is not arbitrary. It gets the prefix @qawolf/ based on our package published name, and the second part is the name given in the rules/index.ts, where you export the rule (see the other comment below).

To combine your comment and mine, I think the rule name will end up being `@qawolf/no-floating-promises`.

Reply:
[Fix name in thorough config](https://github.com/qawolf/eslint-plugin/pull/3/commits/be5477f601725207b37311f40263a48b7a209826)